lib/ex_unit/test/ex_unit/case_test.exs

```diff
@@ -220,3 +220,19 @@ defmodule ExUnit.CaseTest.TmpDir do
     end
   end
 end
+
+defmodule ExUnit.BadOptsCase do
+  use ExUnit.Case, async: true
+
+  test "raises if passed something other than options" do
+    assert_raise ArgumentError, ~r/must be a list of options, got: "not a list of options"/, fn ->
+      Code.eval_quoted(
+        quote do
+          defmodule MyBadCase do
+            use ExUnit.Case, "not a list of options"
+          end
+        end
+      )
```

Comment:
Nice catch! That works. I think the test is small enough that it might be worth keeping. We are never calling `use ExUnit.Case` with something other than options in the codebase, so we're not really testing that we raise if we don't add this test.